package/yast2.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Sep  1 08:08:30 UTC 2022 - Knut Anderssen <kanderssen@suse.com>
+
+- Allow to force an enablement of the selected network service even
```

Comment:
I find the description rather confusing for a user. I would try to think on a higher level.